### PR TITLE
chore(WEB-14): remove dead code and files

### PR DESCRIPTION
**Pull request by Madahva**

### CODE REVIEW

File: fonts/Helvetica.ttf
Comment (nikiburgos) [inline fonts/Helvetica.ttf:1]: Why is this file removed? I'm using it.

Comment (Madahva) [reply fonts/Helvetica.ttf:1]: image

We have two copies. I deleted the one outside the public folder, and everything still works fine. Are you sure the one you're using isn't the one inside the public folder?

Comment (nikiburgos) [reply fonts/Helvetica.ttf:1]: Yes, it works because I didn't merge yet the tailwind code, but I need this file for that

**Review by nikiburgos — APPROVED** [review]
